src/EA.Weee.Core/Scheme/CategoryValue.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace EA.Weee.Core.Scheme
+{
+    using DataReturns;
+    using Helpers;
+    using System;
+
+    [Serializable]
+    public class CategoryValue
```

Comment:
We can just reuse the AATF one, move it to the shared folder

Reply:
removed and replaced with the one in the Shared